src/euphorie/client/browser/risk.py

```diff
@@ -261,18 +261,23 @@ def __call__(self):
         else:
             self._prepare_risk()
             if self.is_custom_risk:
-                template = ViewPageTemplateFile(
-                    "templates/risk_identification_custom.pt"
-                ).__get__(
-                    self, ""
-                )  # noqa
                 next = FindNextQuestion(
                     self.context, self.context.session, filter=self.question_filter
                 )
                 self.has_next_risk = next or False
-            else:
-                template = self.template
-            return template()
+            return self.template()
+
+    @property
+    def template(self):
```

Comment:
Given my previous suggestions are accepted this property can go away

Reply:
But, I _want_ to have this property, so that I can overwrite it in daimler, so that I can consider more use cases and add a 3rd template option.

Your other suggestions are fine. I will try them out & add them.